client/src/components/ProfileDetails/About/About.tsx

```diff
@@ -1,15 +1,39 @@
-import { Avatar, Box, Fade, Grid, GridList, GridListTile, Paper, Typography } from '@material-ui/core';
+import { useEffect, useState } from 'react';
+import { Avatar, Button, Box, Fade, Grid, GridList, GridListTile, Paper, Typography } from '@material-ui/core';
+import { Link } from 'react-router-dom';
 import LocationOnIcon from '@material-ui/icons/LocationOn';
 
 import { Profile } from '../../../interface/Profile';
+import { useAuth } from '../../../context/useAuthContext';
+import { useSnackBar } from '../../../context/useSnackbarContext';
+import { ReviewsApiData } from '../../../interface/ReviewApiData';
+import { getReviews } from '../../../helpers/APICalls/reviews';
 import useStyles from './useStyles';
+import SitterReview from '../Review/Review';
+import CreateReview from '../CreateReview/CreateReview';
 
 export interface Props {
   sitter: Profile | null | undefined;
 }
 
 export default function About({ sitter }: Props): JSX.Element {
+  const [toggle, setToggle] = useState<boolean>(false);
+  const { loggedInUser, sitterReviews, updateReviewsContext } = useAuth();
+  const { updateSnackBarMessage } = useSnackBar();
   const classes = useStyles();
+
+  useEffect(() => {
+    if (sitter) {
+      getReviews(sitter._id).then((data: ReviewsApiData) => {
+        if (data.success) {
```

Comment:
Will it be more efficient to populate reviews array when getting sitters list instead of doing manually every time sitter about page is opened.
This is because we need to show ratings on listings page too.

Reply:
That was my initial thought but if sitter had a few hundred or few thousand reviews each, it would impact overall performance. However, on the implementation side it makes things a lot easier so I have decided to populate reviews rather than fetching them individually.